db/migrations/0007_alter_order_created_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-01-30 18:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0006_alter_order_created_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True),
+        ),
+    ]

```

Comment:
It is a good practice to include one migration per app per pull request in Django projects to not overload the `migrations` folder so that one migration will represent all changes done to models within the app. Please, include one migration.